flaskerizer.py

```diff
@@ -1,8 +1,10 @@
 import os
 from structure_directory import StructureDirectory
 from write_app import WriteApp
+import config
```

Comment:
Imports usually go by alphabetical order by convention, its not a deal breaker though. I miss it sometimes